test/run_tests.pl.in

```diff
@@ -30,6 +30,23 @@ my $cmd;
 my $output;
 my $status = 0;
 my $myprefix;
+my $mypathstr;
+my $mymcapaths;
+my @myfullpaths;
+my $mybuilddir;
+
+$mybuilddir = "@PMIX_BUILT_TEST_PREFIX@";
+
+$mypathstr = "@PMIX_COMPONENT_LIBRARY_PATHS@";
+
+my @splitstr = split(':', $mypathstr);
+
+foreach my $path (@splitstr) {
+    my $fullpath = $mybuilddir . "/" . $path;
+    push(@myfullpaths, $fullpath);
+}
+$mymcapaths = join(":", @myfullpaths);
+$ENV{'OMPI_MCA_mca_base_component_path'} = "$mymcapaths";
```

Comment:
Should that be `OMPI_MCA` or `PMIX_MCA`?

Reply:
Yeah, I caught that and fixed it, and this works fine...except it fails for me in "make_dist_tarball". The issue it flags is that it cannot find an acceptable plugin for some of the frameworks, even though they are included in the component_path list. I thought at first that the envar might be too long (there are a lot of components and the paths are long), but pruning it didn't solve the problem. I thought about pointing the path at the ```_inst``` location instead of ```_build```, but haven't had time to try it yet.

Reply:
Just for clarity, Mellanox Jenkins CI is also executing "make_dist_tarball" - but that is succeeding because it has ```-disable-dlopen``` hard-coded in it.

Reply:
Hmm...added some preliminary work on the new approach and now Mellanox fails. Checking to see why - can't get at the test-suite.log, sadly.